setup.py

```diff
@@ -18,7 +18,7 @@
     description='httplib2 caching for requests',
     long_description=long_description,
     install_requires=[
-        'requests',
+        'requests==2.12.4',
```

Comment:
We need to try and support as many versions of requests as possible, so this shouldn't be pinned. As an addon to requests, I consider it CacheControl's role to support the upstream requests dependency rather than define it.